src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -1855,6 +1855,19 @@ public object ConvertBack(object value, Type targetType, object parameter, Cultu
         }
     }
 
+    public class PythonSelectionToStringConverter : IValueConverter
```

Comment:
This converter is pretty generic, so we should pick a better name so it's easy to tell it's reusable. How about `CompareToParameterConverter`?